### ISSUE TITLE
Sanitize view names

### ISSUE THREAD

**Issue by Neraste** [DakaraProject/dakara-server#40]
Use a better organization of views' name in the project. This will only affect views files and `url.py`.

Also, sanitize `url.py` organization by removing starred imports and following PEP8.

### Playlist

Split front views and player views in separate view files, but import them in the playlist view:

```python
# view.py
from . import view_front as front
from . import view_player as player
```

Changes views' name, in `view_front.py`:

* `PlaylistEntryDetail` → `PlaylistEntryView`;
* `PlaylistEntryList` → `PlaylistEntryListView`;
* `PlayerForUserView` → `PlayerStatusView`;
* `PlayerCommandForUserView` → `PlayerManageView`;
* `PlayerErrorsForUserView` → `PlayerErrorsPoolView`;
* `PlayerDetailsCommandErrorsForUserView` → `PlayerView`

In `view_player.py`:

* `PlayerForPlayerView` → `PlayerView`;
* `PlayerErrorForPlayerView` → `PlayerErrorView`.

### Library

Just change views' name:

* `SongList` → `SongListView`;
* `SongDetailView` → `SongView`;
* `ArtistList` → `ArtistListView`;
* `WorkList` → `WorkListView`;
* `WorkTypeList` → `WorkListTypeView`.

### Users

Just change views' name:

* `CurrentUser` → `CurrentUserView`;
* `UserList` → `UserListView`;
* `UserView`;
* `PasswordView`.

**Comment by Nadeflore** [comment]
:+1:

**Comment by Nadeflore** [comment]
Resolved in #43
